### PR TITLE
php8 | support laminas-form@3.0.0

### PR DESCRIPTION
**Pull request by stefanvandekaa**
Signed-off-by: Stefan Van de Kaa <stefan@trilectica.nl>

I use laminas-form in my code with ocramius/proxy-manager and laminas-code.
In order to use them all correctly this plugin should be compatible with laminas-form 3.0.x

Therefore i upgrade the package. Only file which is used from this package is the file:

https://github.com/laminas/laminas-form/blob/3.1.x/src/FormInterface.php

Which has no backward break with 2.7 therefore all should work and an upgrade would be possible.

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (froschdesign) [inline .idea/workspace.xml]: Please remove this file from the pull request. (You can add the `.idea` directory to your local Git exclude file.)
Thanks in advance! 👍

Comment (stefanvandekaa) [reply .idea/workspace.xml]: @froschdesign sorry, that shouldnt be committed. :(

**Comment by stefanvandekaa** [comment]
Is there anything to do here, or any way to get this merged?

**Comment by froschdesign** [comment]
@stefanvandekaa 
_Sorry for the late response, I was busy with client projects._

> Is there anything to do here, or any way to get this merged?

Can you check the failed checks again?

**Comment by adelarcubs** [comment]
Hello @froschdesign 

I need this chenge too kkk. Looking others PRs, I saw the same error and those PRs was accepted.

Could you recheck this PR? And about the error, I'll look a solution in other PR :)

Cheers

**Comment by adelarcubs** [comment]
My bad, the PR was not merged...

**Comment by weierophinney** [comment]
Closing in favor of #18, which provides full support for PHP 8.0 and 8.1, and updates to laminas-form v3 releases.

**Pull request closed without merging**